### PR TITLE
account-email-changes update

### PR DESCRIPTION
**Pull request by creativeappdevamit**
# What and why?

# How to test?

# Trello
https://trello.com/c/qHvQrOWt/512-s48-user-account-creation-change-own-email-address

### CODE REVIEW

**Review by mybigopenbox — APPROVED** [review]

File: frontstage/templates/account/account-change-email-address.html [hunk 1]
```diff
@@ -31,20 +31,39 @@
...
+            {% endif %}
+        {% endfor %}
     {% endif %}
 {% endwith %}
 <h1 class="u-fs-xl">Change email address</h1>
 <p>You will need to authorise a change of email address.</p>
 <p>We will send a confirmation email to <b>{{ new_email }}</b>.</p>
+{% call onsPanel({}) %}
+    <div>
+        If you are longer required to respond to your surveys, you can <a href="{{ url_for('account_bp.transfer_survey_overview') }}">transfer your surveys</a>to a colleague.
```
Comment (insacuri) [inline frontstage/templates/account/account-change-email-address.html:64]: there needs to be a space after the </a>

**Review by insacuri — CHANGES_REQUESTED** [review]
That comment, and also if you change the email address to nothing and submit then it lets you try and change your email address to something blank

File: frontstage/templates/account/account-change-email-address.html (hunk 1 above)
Comment (creativeappdevamit) [reply frontstage/templates/account/account-change-email-address.html:64]: 👍

**Review by insacuri — APPROVED** [review]

**Pull request merged**